content/it/api-gateway.md

```diff
@@ -0,0 +1,17 @@
+---
+title: API Gateway
+status: Completed
+category: tecnologia
+tags: ["networking", "", ""]
+---
+
+## Cos'è  
+Un [API](/it/application-programming-interface/) Gateway è uno strumento che aggrega API applicative rendendole disponibili in un'unica posizione. Questo consente alle organizzazioni di spostare le funzioni chiave, come l'autenticazione e l'autorizzazione o la limitazione del numero di richieste tra le applicazioni, in una posizione gestita centralmente. Un API Gateway fornisce inoltre un'unica interfaccia comune per i consumatori di API (spesso esterni).
```

Comment:
ho visto che in inglese è effettivamente così, ma io questo link lo sposterei al chiaro riferimento alle api
```suggestion
Un API Gateway è uno strumento che aggrega [API](/it/application-programming-interface/) applicative rendendole disponibili in un'unica posizione. Questo consente alle organizzazioni di spostare le funzioni chiave, come l'autenticazione e l'autorizzazione o la limitazione del numero di richieste tra le applicazioni, in una posizione gestita centralmente. Un API Gateway fornisce inoltre un'unica interfaccia comune per i consumatori di API (spesso esterni).
```

Reply:
Si, correttissimo.
Grazie!